### ISSUE TITLE
Task :react-native-video:verifyReleaseResources FAILED

### ISSUE THREAD

**Issue by ndrkltsk** [TheWidlarzGroup/react-native-video#1467]
### Current behavior
The error happens on building for android, only release, debug is ok.

### package.json
```
"dependencies": {
    "react": "16.3.1",
    "react-native": "0.58.3",
    "react-native-video": "^4.3.1",
    ...
  }
```

### app build.gradle
```

android {
    compileSdkVersion 28
    buildToolsVersion '28.0.3'

    defaultConfig {
        minSdkVersion 22
        targetSdkVersion 28
        ndk {
            abiFilters "armeabi-v7a", "x86"
        }
       ...
    }
  
    ....

    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_8
        targetCompatibility JavaVersion.VERSION_1_8
    }

}

dependencies {
    implementation project(':react-native-video')
    implementation fileTree(dir: "libs", include: ["*.jar"])
    implementation "com.android.support:appcompat-v7:28.0.0"
    implementation "com.facebook.react:react-native:0.58.3"
    ...
}
```

### Logs
```
> Task :app:bundleReleaseJsAndAssets
warning: the transform cache was reset.
Loading dependency graph, done.
bundle: Writing bundle output to: /home/andriy/workspace/react-native/old_app_vitalitys/android/app/build/generated/assets/react/release/index.android.bundle
bundle: Done writing bundle output
bundle: Copying 321 asset files
bundle: Done copying assets

> Task :react-native-video:verifyReleaseResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-video:verifyReleaseResources'.
> java.util.concurrent.ExecutionException: com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed
  /home/andriy/workspace/react-native/old_app_vitalitys/node_modules/react-native-video/android-exoplayer/build/intermediates/res/merged/release/values/values.xml:279: error: resource android:attr/fontVariationSettings not found.
  /home/andriy/workspace/react-native/old_app_vitalitys/node_modules/react-native-video/android-exoplayer/build/intermediates/res/merged/release/values/values.xml:280: error: resource android:attr/ttcIndex not found.
  error: failed linking references.

```

**Comment by mmalek95** [comment]
I'm facing same issue, any updates?

**Comment by shlokamin** [comment]
Same :/

**Comment by adawi9** [comment]
Did any one solve this?

**Comment by adawi9** [comment]
I solved my problem by adding this

Please follow  [this link](https://reactnativeforyou.com/aapt-error-resource-androidattr-colorerror-not-found-react-native-error-fix/)

` subprojects {
        afterEvaluate {
            project ->
                if (project.hasProperty("android")) {
                    android {
                        compileSdkVersion = 28
                        buildToolsVersion = "28.0.3"
                    }
                }
        }
    }`

**Comment by art1373** [comment]
I am still having issues with this
rn version : 0.61.4

**Comment by art1373** [comment]
> I am still having issues with this
> rn version : 0.61.4
fixed npm i react-native-video@5.0.0

**Comment by vipulchasta** [comment]
### Following two changes saved my day 😅.


**1. Updated "react-native-video-controls" Module**
> "react-native-video-controls": "^2.4.0" => "react-native-video-controls": "^2.6.0"



**2. Some modification in /android/settings.gradle**
Used 
> project(':react-native-video').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-video/android')

instead of
> project(':react-native-video').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-video/android-exoplayer')
